### ISSUE TITLE
[BUG] Volume's name is not required

### ISSUE THREAD

**Issue by lanfon72** [harvester/harvester#7030]
### Describe the bug
User can create volume without input any name.

https://github.com/user-attachments/assets/00fc7ac3-3fcc-4b0f-a40f-66a8eebc142a

### To Reproduce
1. Install Harvester with any nodes
2. Navigate to Volumes and create new volume with only input `size`

### Expected behavior
**name** is required field in `v1.3.2`, if it is a breaking change, we should:
1. Update related documents
2. Update _required field_ in UI
![image](https://github.com/user-attachments/assets/9d614cdb-3a14-419d-877d-4bf0a02a9dcd)

## Environment:
 - Underlying Infrastructure (e.g. Baremetal with Dell PowerEdge R630): **Baremetal DL360G9 single node**
 - Harvester ISO version: **v1.4-bede7c36-head**
 - **ui-source** Option: **Auto**

**Comment by bk201** [comment]
@FrankYang0529 is this the result of the underlying framework update?

**Comment by FrankYang0529** [comment]
> @FrankYang0529 is this the result of the underlying framework update?

Yes, it's steve server change. I think we can close this issue. If we want to prevent it, we may need to add the check in the UI. It's steve server default API, so harvester pod can't catch the request.

https://github.com/harvester/harvester/pull/6666#issuecomment-2409626314

**Comment by bk201** [comment]
@lanfon72 is it OK to close the issue? I understand it's a behavior change, but it looks like generating a name when the user doesn't provide it is no harm in most cases.

**Comment by lanfon72** [comment]
> @lanfon72 is it OK to close the issue? I understand it's a behavior change, but it looks like generating a name when the user doesn't provide it is no harm in most cases.

If so, we would need update UI to remove name is required.

![image](https://github.com/user-attachments/assets/19abdd39-d1a5-42ba-be39-11b796d5a6b8)

**Comment by a110605** [comment]
> If so, we would need update UI to remove name is required.
> 
> ![image](https://github.com/user-attachments/assets/19abdd39-d1a5-42ba-be39-11b796d5a6b8)

https://github.com/harvester/harvester-ui-extension/pull/274 remove required dot from UI.

**Comment by noahgildersleeve** [comment]
Validated in Harvester `v1.6.0-rc1` and upstream Rancher `v2.12.0-alpha17` with ui-extensions `v1.6.0-rc1`.

<img width="2543" height="607" alt="Image" src="https://github.com/user-attachments/assets/66b8cdbb-7b61-42bf-8e80-7eac8f0083b7" />
<img width="2541" height="605" alt="Image" src="https://github.com/user-attachments/assets/1b550cd7-5641-4023-82a8-363854dc159f" />
